geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureReader.scala

```diff
@@ -89,7 +90,8 @@ class AccumuloFeatureReaderWithStats(query: Query,
 
   override protected def closeOnce(): Unit = {
     iter.close()
-    val stat = QueryStat(qp.sft.getTypeName,
+    val stat = QueryStat("Accumulo Vector",
```

Comment:
As we use the string, "Accumulo Vector" in a number of places, should we pull this out into a constant?